modules/mri_violations/php/mri_protocol_violations.class.inc

```diff
@@ -90,17 +90,17 @@ class Mri_Protocol_Violations extends \NDB_Menu_Filter
                                'Series_Description',
                                'Minc_Location',
                                'PatientName',
-                               'TR_Range',
-                               'TE_Range',
-                               'TI_Range',
-                               'Slice_Thickness_Range',
-                               'Xspace_Range',
-                               'Yspace_Range',
-                               'Zspace_Range',
-                               'Xstep_Range',
-                               'Ystep_Range',
-                               'Zstep_Range',
-                               'Time_Range',
+                               'TR',
```

Comment:
so you opted to change the name of the columns in the front end but kept them the same in the back end?
I think this is fine; maybe we can have a separate (cleanup) redmine ticket to change those in the backend as well?
@driusan is what I am suggesting worth the effort or we can just keep the table untouched?